lingui.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  locales: ['en', 'es', 'pseudo'],
```

Comment:
Had to remove, otherwise it would add links like cow.fi/es/tokens....

Reply:
Ok.... for now, until @avsavsavs or I translate into 🇪🇸

Reply:
¡Vale, perfecto!